### ISSUE TITLE
Use as start for a new Delta-CRAMPS

### ISSUE THREAD

**Issue by mgiaco** [strahlex/MendelMax-CRAMPS#2]
Hello Alex,

What do you think is this a good idea to use this for a new Delta config?
That´s the pythonic way right?

thanks

**Comment by machinekoder** [comment]
Good idea, basically you can start of by creating a new function for setting up the stepper motors for the delta config: https://github.com/strahlex/MendelMax-CRAMPS/blob/master/config/base.py#L57

**Comment by mgiaco** [comment]
Hello Alex, 

Okay, but why do you mean I need a new function for setting up the steppers? I will follow this setup https://github.com/artekw/MiniKossel-CRAMPS. It is basically what I want and it must be a fork of yours. But can you give me a hint where I can set the pru_periode. I do need a faster setup? 

I also tried this setup https://github.com/Arcus-3d/Arcus-3D-M1 but it does not use the python lib to init the HAL. But strange whit this setup is that if I load the Machineface it will hang sometimes and also there are some widgets which I cannot see anymore after this interrupt.
